styx-client/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java

```diff
@@ -206,6 +206,43 @@ public void shouldCreateBackfill() throws Exception {
     assertThat(request.method(), is("POST"));
   }
 
+  @Test
+  public void shouldCreateBackfillFromInput() throws Exception {
+    final Instant start = Instant.parse("2017-01-01T00:00:00Z");
+    final Instant end = Instant.parse("2017-01-30T00:00:00Z");
+    final BackfillInput backfillInput = BackfillInput.newBuilder()
+        .start(start)
+        .end(end)
+        .component("foo-comp")
+        .workflow("bar-wf")
+        .concurrency(1)
+        .reverse(true)
+        .build();
+
+    final Backfill backfill = Backfill.newBuilder()
```

Comment:
The values for `backfillInput` and this `backfill` seem magic. Either extract them, or use them once and then reuse them, like done here https://github.com/spotify/styx/pull/455/files?utf8=%E2%9C%93&diff=split#diff-ae03c6a48f8fa80596ad167c226fdde5R272